app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
```

Comment:
@barnabyalter ,
Sure, I can do that. I wasn't sure if there would be a use case, that's why I didn't want to mess with existing functions. I was wondering if "450" should be hard coded though. Should I keep it as a variable somewhere?